epersons.md

```diff
@@ -74,3 +127,58 @@ the replace operation `[{ "op": "replace", "path": "/password", "value": "newpas
   "password": "newpassword",
 ```
 NOTE: The new password is currently returned after an update but this could be revisited later, see [#30]((https://github.com/DSpace/Rest7Contract/issues/30))
+
+## Create new EPerson
+
+**POST /api/eperson/epersons**
+
+To create a new EPerson, perform a post with the JSON below to the epersons endpoint when logged in as admin.
+
+```json
+{
+  "name": "user@institution.edu",
+  "metadata": {
+    "eperson.firstname": [
+      {
+        "value": "John",
+        "language": null,
+        "authority": "",
+        "confidence": -1
+      }
+    ],
+    "eperson.lastname": [
+      {
+        "value": "Doe",
+        "language": null,
+        "authority": "",
+        "confidence": -1
+      }
+    ]
+  },
+  "canLogIn": true,
+  "email": "user@institution.edu",
+  "requireCertificate": false,
+  "selfRegistered": true,
+  "type": "eperson"
+}
+```
+
```

Comment:
if you have time it would be useful to add the error code here explaining that the endpoint can be used only by administrators, otherwise we can improve that later